scripts/scil_surface_create.py

```diff
@@ -0,0 +1,170 @@
+                           binary_fill_holes)
+import mcubes
+
+from scilpy.image.labels import (get_data_as_labels,
+                                 merge_labels_into_mask)
+from scilpy.io.image import get_data_as_mask
+from scilpy.io.utils import (add_overwrite_arg,
+                             add_verbose_arg,
+                             assert_inputs_exist,
+                             assert_outputs_exist,
+                             ranged_type)
+
+EPILOG = """
+References:
+[1] St-Onge, E., Daducci, A., Girard, G. and Descoteaux, M. 2018.
+    Surface-enhanced tractography (SET). NeuroImage.
+"""
+
+
+def _build_arg_parser():
+    p = argparse.ArgumentParser(description=__doc__, epilog=EPILOG,
+                                formatter_class=argparse.RawTextHelpFormatter)
+
+    g = p.add_argument_group("Input (Labels or Mask)")
+    mxg = g.add_mutually_exclusive_group(required=True)
+    mxg.add_argument('--in_labels',
+                     help='Path of the atlas (nii or nii.gz). '
+                     'It will use the indices provided with --indices. '
+                     'If no indices are provided, it will use all indices.')
+    mxg.add_argument('--in_mask',
+                     help='Path of the mask (nii or nii.gz).')
+
+    p.add_argument('out_surface',
+                   help='Output surface (.vtk)')
+
+    p.add_argument('--indices', nargs='+',
+                   help='List of labels indices to use for the surface.')
+
+    p.add_argument('--value', default=0.5,
+                   type=ranged_type(float, 0, None, min_excluded=False),
+                   help='Isosurface threshold value used. '
+                        'This value is called isovalue '
+                        'in mbcube. [%(default)s]')
+
+    morpho_g = p.add_argument_group('Morphology options')
+    morpho_g.add_argument('--smooth',
+                          type=ranged_type(float, 0, None, min_excluded=False),
+                          help='Smoothing size with'
+                               ' 1 implicit step. [%(default)s]')
+    morpho_g.add_argument('--erosion',
+                          type=ranged_type(int, 0, None, min_excluded=False),
+                          help='Erosion: number of iterations. [%(default)s]')
+    morpho_g.add_argument('--dilation',
+                          type=ranged_type(int, 0, None, min_excluded=False),
+                          help='Dilation: number of iterations. [%(default)s]')
+    morpho_g.add_argument('--opening',
+                          type=ranged_type(int, 0, None, min_excluded=False),
+                          help='Opening (dilation of the erosion): number '
+                               'of iterations. [%(default)s]')
+    morpho_g.add_argument('--closing',
+                          type=ranged_type(int, 0, None, min_excluded=False),
+                          help='Closing (erosion of the dilation): number '
+                               'of iterations. [%(default)s]')
+
```

Comment:
--max_label   option was removed,
Maybe this was wanted, it was useful to remove spurious voxel from a mask 
(was not use for SET)

Reply:
I think this could/should be done outside of the script.